src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[random.nextInt(Color.values().length)].name();
```

Comment:
It's correct, but I suggest to split such complicated lines into multiple lines. It would be easier to read
```suggestion
        int randomColorIndex = random.nextInt(Color.values().length);
        return Color.values()[randomColorIndex ].name();
```